tests/opentracing_instrumentation/test_mysqldb.py

```diff
@@ -0,0 +1,78 @@
+import sys
+
+import pytest
+
+from opentracing.ext import tags
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from opentracing_instrumentation.client_hooks import mysqldb as mysqldb_hooks
+from opentracing_instrumentation.request_context import span_in_context
+from .sql_common import metadata, User
+
+
+SKIP_REASON_PYTHON_3 = 'MySQLdb is not compatible with Python 3'
+SKIP_REASON_CONNECTION = 'MySQL is not running or cannot connect'
+POSTGRES_CONNECTION_STRING = 'mysql://root@127.0.0.1/test'
```

Comment:
MYSQL_CONNECTION_STRING ?

Reply:
Yes, it definitely should be `MYSQL_CONNECTION_STRING`.
Thank you!